android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -69,6 +72,13 @@ object Rownd {
                     }
             }
         }
+        coroutineScope = CoroutineScope(Dispatchers.IO).launch {
```

Comment:
There's probably a better method of updating the RowndCustomizations but I wasn't sure how. Adding the Rownd.state.value directly to the RowndCustomization class didn't provide the most up to date state data.

Reply:
The serializers for customStylesFlag and fontFamily has been added. For whatever reason I could not get an optional value to work for Boolean and String. Whenever I set the initial value to null, it would only return null